alicloud/resource_alicloud_lindorm_instance_test.go

```diff
@@ -0,0 +1,442 @@
+package alicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aliyun/terraform-provider-alicloud/alicloud/connectivity"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccAlicloudLindormInstance_basic0(t *testing.T) {
+	var v map[string]interface{}
+	resourceId := "alicloud_lindorm_instance.default_0"
+	ra := resourceAttrInit(resourceId, AlicloudLindormInstanceMap0)
+	rc := resourceCheckInitWithDescribeMethod(resourceId, &v, func() interface{} {
+		return &HitsdbService{testAccProvider.Meta().(*connectivity.AliyunClient)}
+	}, "DescribeLindormInstance")
+	rac := resourceAttrCheckInit(rc, ra)
+	testAccCheck := rac.resourceAttrMapUpdateSet()
+	rand := acctest.RandIntRange(10000, 99999)
+	name := fmt.Sprintf("tf-testaccLindorminstance%d", rand)
+	testAccConfig := resourceTestAccConfigFunc(resourceId, name, AlicloudLindormInstanceBasicDependence0)
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+		},
+		IDRefreshName: resourceId,
+		Providers:     testAccProviders,
+		CheckDestroy:  rac.checkResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"disk_category":             "cloud_efficiency",
+					"payment_type":              "POSTPAY",
+					"zone_id":                   "${local.zone_id}",
+					"vswitch_id":                "${local.vswitch_id}",
+					"instance_name":             "${var.name}",
+					"file_engine_specification": "lindorm.c.xlarge",
+					"file_engine_node_count":    "2",
+					"instance_storage":          "480",
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"disk_category":             "cloud_efficiency",
+						"payment_type":              "POSTPAY",
+						"instance_name":             name,
+						"file_engine_specification": "lindorm.c.xlarge",
+						"file_engine_node_count":    "2",
+						"instance_storage":          "480",
+					}),
+				),
+			},
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"ip_white_list": []string{"118.118.118.118"},
+				}),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheck(map[string]string{
+						"ip_white_list.#": "1",
+					}),
+				),
+			},
+			{
+				Config: testAccConfig(map[string]interface{}{
+					"instance_name": "tf-acc-test-instance-new",
```

Comment:
The name should be prefixed with tf-testAcc. Here can use `name +"update" instead.